dumb-jump.el

```diff
@@ -1036,6 +1036,14 @@ or most optimal searcher."
                    "functor test (T:TEST) ="
                    "functor test(T:TEST) ="))
 
+    ;; sql
+    (:type "function" :supports ("ag" "grep" "rg" "git-grep") :language "sql"
+           :regex "(CREATE|create)\\s+(.+?\\s+)?(FUNCTION|function|PROCEDURE|procedure)\\s+JJJ\\s*\\\("
```

Comment:
I hate having to define the two variants, but couldn't find support for declaring the regex as case-insensitive. Perhaps adding an additional key for additional options to be passed to the command would do?